backend/inspirehep/search/facets.py

```diff
@@ -177,6 +178,29 @@ def inner(values):
     return inner
 
 
+def curation_filter():
+    filter_for_option = {
+        "Claimed papers": {
+            "nested": {
+                "path": "authors",
+                "query": {"term": {"authors.curated_relation": True}},
+            }
+        },
+        "Unclaimed papers": {
+            "nested": {
+                "path": "authors",
+                "query": {"term": {"authors.curated_relation": False}},
+            }
+        },
+    }
```

Comment:
* not sure this part is correct, as you need to produce a `nested` selecting the author and the claim status together (if you have two separate `nested`, you're doing a selection based on claims for any authors here).
* also, because `curated_relation` is not mandatory, it might simply be absent, so better do a search for                                        `not True` instead of `False`.